docs/basic-features/fast-refresh.md

```diff
@@ -77,6 +77,7 @@ local state being reset on every edit to a file:
 - Sometimes, a file would export the result of calling higher-order component
   like `HOC(WrappedComponent)`. If the returned component is a
   class, state will be reset.
+- Anonymous arrow functions like `export default () => <div />;` cause Fast Refresh to not preserve local component state.
```

Comment:
Could you include a code sandbox showing this? I could be wrong, but I thought I've seen this work fine which using `useState` inside an arrow function.

Reply:
Interesting. It looks like you should be providing a name anyway, because Next is going to throw an error. 

https://github.com/vercel/next.js/issues/12891#issuecomment-628846848

Reply:
Related: https://github.com/vercel/next.js/issues/13024

Reply:
It seems to overlap with this description. however, it is not in the fast refresh document, can I add it?

https://nextjs.org/docs/advanced-features/codemods#name-default-component

Reply:
@leerob https://codesandbox.io/s/trusting-chandrasekhar-m4zi8?file=/pages/index.js
Please check the console.